docs/zh/UI/quickStart/image.md

```
@@ -0,0 +1,48 @@
+label: UI
+---
+
+`Image` 组件用于在 `UICanvas` 中显示图片。
+
+## 编辑器使用
+
+### 添加 Image 节点
+
+在 **[层级面板](/docs/interface/hierarchy/)** 添加 `Image` 节点
+
+<img src="https://mdn.alipayobjects.com/huamei_yo47yq/afts/img/A*9SCNTZNglo0AAAAAAAAAAAAAehuCAQ/original" style="zoom:50%;" />
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)